### PR TITLE
Extends assert_near to date comparison

### PR DESCRIPTION
**Pull request by benjello**
#### Technical changes

- Extends assert_near to date comparison

### CODE REVIEW

**Comment by benjello** [comment]
@sandcha : I do nt understand the test erros since it seems unrelated to my changes. any hints @sandcha ?

**Comment by sandcha** [comment]
Hi @benjello , As the errors in thrown on this test: 
```py
def test_parameter_node():
        response = subject.get('/parameter/benefits')
...
```
and the last update of the openfisca/country-template#99 adds a `openfisca_country_template/parameters/benefits/parenting_allowance.yaml`, which comes with a minor update of the package, I looks like a test that is a close to the implementation and that we can update to avoid this type of situation.

**Comment by benjello** [comment]
@sandcha : tests pass now ! You can review ;-)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 35.1.0 [#973](https://github.com/openfisca/openfisca-core/pull/973)
```
Comment (sandcha) [inline CHANGELOG.md:3]: Just a reminder: bump needed (+ only 2 `#`)

**Review by sandcha — APPROVED** [review]
LGTM. Thx @benjello! 
Please don't forget to update with a `rebase` and, in the best of all possible worlds, set the name of the PR as message for the merging commit :)

**Pull request merged**